src/board/ui.ts

```diff
@@ -18,12 +19,32 @@ export function createBoard(
 ) {
   document.body.insertAdjacentHTML("afterbegin", svgText);
   const svg = document.querySelector("svg");
-  // We start stopped.
   if (!svg) {
     throw new Error("No SVG");
   }
-  svg.style.opacity = stoppedOpactity;
-  return new BoardUI(operations, fs, svg, onSensorChange);
+  const playButton = createPlayButton();
+  svg.insertAdjacentElement("afterend", playButton);
+  return new BoardUI(operations, fs, svg, playButton, onSensorChange);
+}
+
+function createPlayButton() {
+  const playButton = document.createElement("button");
+  playButton.style.position = "absolute";
+  playButton.style.display = "flex";
+  playButton.style.justifyContent = "center";
+  playButton.style.alignItems = "center";
+  playButton.style.width = "100%";
+  playButton.style.height = "100%";
+  playButton.style.top = "0";
+  playButton.style.left = "0";
+  playButton.style.backgroundColor = "transparent";
+  playButton.style.border = "none";
+  playButton.style.cursor = "pointer";
```

Comment:
It might be nicer to define a class in simulator.html ?

Reply:
My brain was stuck in 'everything in JavaScript' mode... [`72b1746`](https://github.com/microbit-foundation/micropython-microbit-v2-simulator/pull/23/commits/72b174643bcaed00b861fb42fb9d32eced1cd41d).